lexic.txt

```diff
@@ -1,37 +0,0 @@
-Alphabet:
-
-a. Upper (A-Z) and lower case letters (a-z) of the English alphabet
-b. Underline character '_';
-c. Decimal digits (0-9);
-
--operators + - * / < <= = >= eq mod ** not
-
-- separators [ ] { } ; space
-
-- reserved words: init verify otherwise escape true untrue while for func in
-
-integer:
-
-<int>::= 0 | <nonzero>
-<nonzero>::=<nonzerodigit>|-<nonzerodigit>|<nonzerodigit><digitseq>|-<nonzerodigit><digitseq>
-<digitseq>::= <digit>|<digit><digitseq>
-<digit>::=0|<nonzerodigit>
-<nonzerodigit>::=1...
```

Comment:
I suppose you were too lazy to write all the non-zero digits, but could use some better styles like
1 | 2 | ... | 9

Reply:
changed